### DISCUSSION TITLE
Trying to mimic the demo in local results in an empty pdf file.

### DISCUSSION THREAD

**Discussion by etiennelacoursiere** [gotenberg/gotenberg#646] [Q&A]
Hi! 

I'm sure I'm doing something wrong but i don't see what.

When i run the live demo like this

```
curl \
--request POST 'https://demo.gotenberg.dev/forms/chromium/convert/url' \
--form 'url="https://sparksuite.github.io/simple-html-invoice-template/"' \
-o my.pdf
```

It creates a `my.pdf` file as expected with the template in it. 

But when i boot my own container like this `docker run --rm -p 3000:3000 gotenberg/gotenberg:7` and try the same curl but with localhost instead like this: 

```
curl \
--request POST 'https://localhost:3000/forms/chromium/convert/url' \
--form 'url="https://sparksuite.github.io/simple-html-invoice-template/"' \
-o my.pdf
```

It results in the `my.pdf` file being empty.

I tried booting my container with these flag but nothing seems to work 
```
docker run --rm -p 3000:3000 gotenberg/gotenberg:7 \
             gotenberg \
             --chromium-allow-file-access-from-files=true \
             --chromium-allow-insecure-localhost=true \
             --chromium-disable-web-security=true
```

Any idea? Thanks!

**Comment by gulien** [comment]
Hello @etiennelacoursiere,

What’s the exact version of Gotenberg you’re running locally?

Are your network settings allowing your Gotenberg instance to navigate to the given url?

**Reply by etiennelacoursiere** [reply]
My problem was the port. I changed it to something else `docker run --rm -p 3000:3092 gotenberg/gotenberg:7
` and now it's working fine!
